representatives/contrib/francedata/variants.py

```diff
@@ -164,19 +239,29 @@ def _get_an_comittees(json):
                 "start": "%(mandat_debut)s"
             },
             {
-                "kind": "comittee",
+                "kind": "committee",
```

Comment:
The function name still has "comittee" instead of "committee", perhaps some sed incantation could help sort that up to be consistent ?